src/components/header.js

```diff
@@ -56,6 +68,16 @@ const NavItem = styled.div`
   .astropy-link {
     margin-left: auto;
   }
+
+  @media screen and (max-width: 600px) {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-top: 10px;
+    padding-left: 3rem;
+    ${'' /* Or .5 rem for the left padding */}
```

Comment:
Yeah I think I like `padding-left: 0.5 rem` to line up with the logo rather than lining up on "Learn.Astropy."